docs/MVS_parameters.rst

```diff
@@ -1,137 +1,817 @@
+.. _country-label:
 
-**scenario_id**: Users can assign a scenario id as per their preference.
+country
+********
 
-**scenario_name**: Users can assign a scenario name as per their preference.
+:Definition: Name of the country where the project is being deployed
+:Type: str
+:Unit: None
+:Example: Norway
+:Restrictions: None
+:Default: None
 
+|
 
-economic_data.csv
------------------
+.. _currency-label:
 
-**currency**: The currency of the country where the project is implemented. For example, in the case of Germany, the value is EUR. For Norway, it is NOK. 
+currency
+************
 
-**discount_factor**: Discount factor is the factor which accounts for the depreciation in the value of money in the future, compared to the current value of the same money. The common method is to calculate the weighted average cost of capital (WACC) and use it as the discount rate.
+:Definition: The currency of the country where the project is implemented.
+:Type: str
+:Unit: None
+:Example: EUR (if the country is Germany)
+:Restrictions: None
+:Default: None
 
-**Project_duration**: The name of years the project is intended to be operational. The project duration also sets the installation time of the assets used in the simulation. After the project ends these assets are 'sold' and the refund is charged against the initial investment costs.
+|
 
-**tax**: Tax factor. 
+.. _crate-label:
 
-simulation_settings.csv
------------------------
+c-rate
+*********
 
-**display_output**: [Developer setting] Default value is "debug"
+:Definition: C-rate is the rate at which the storage can charge or discharge relative to the nominal capacity of the storage. A c-rate of 1 implies that the battery can discharge or charge completely in a single timestep.
+:Type: Numeric
+:Unit: factor of total capacity (kWh)
+:Example: *storage capacity*: NaN, *input power*: 1, *output power*: 1
+:Restrictions: Only the columns "input power" and "output power" require a value, in column "storage capacity" c_rate should be set to NaN.
+:Default: None
 
-**evaluated_period**: The number of days for which the simulation is to be run.
+|
 
-**output_lp_file**: Acceptable values are either True or False. Entering True would result in the generation of a file with the linear equation system describing the simulation, ie., with the objective function and all the constraints. This lp file enables the user to peer ‘under the hood’ to understand how the program optimizes for the solution.
+.. _developmentcosts-label:
 
+development_costs
+********************
+
+:Definition: A fixed cost to implement the asset, eg. planning costs which do not depend on the (optimized) asset capacity.
+:Type: Numeric
+:Unit: currency
+:Example: 10000
+:Restrictions: None
+:Default: None
+=======
 **start_date**: The data and time on which the simulation starts at the first step. Acceptable format is YYYY-MM-DD HH:MM:SS. E.g.: 2018-01-01 00:00:00
```

Comment:
Do you also still have **start_date** as an entry?

Reply:
Deleted now! :)